src/automation/azext_automation/aaz/latest/automation/python3package/_create.py

```diff
@@ -0,0 +1,305 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation python3package create",

```

Comment:
```suggestion
    "automation python3-package create",

```

Reply:
Why are there two sets of commands `python3package` and `python3-package`?

Reply:
Removed python3package commands